phi/model/google/gemini.py

```diff
@@ -267,7 +267,7 @@ def invoke(self, messages: List[Message]):
         Returns:
             GenerateContentResponse: The response from the model.
         """
-        return self.get_client().generate_content(contents=self._format_messages(messages))
+        return self.get_client().generate_media(contents=self._format_messages(messages))
```

Comment:
@dirkbrnd wont this break Gemini for text generation?

Reply:
same for vertex ai

Reply:
I don't think I made this change? I'll undo anyway

Reply:
Sorry, that was unintentional